google/resource_compute_disk_test.go

```diff
@@ -345,29 +345,43 @@ resource "google_compute_disk" "foobar" {
 }`, diskName)
 }
 
-func testAccComputeDisk_fromSnapshot(firstDiskName, snapshotName, diskName, xpn_host string, ref_selector string) string {
+func testAccComputeDisk_fromSnapshot(projectName, firstDiskName, snapshotName, diskName, org, billingId, ref_selector string) string {
```

Comment:
Why was this test using the XPN project in the first place instead of the regular test project?

Reply:
Not sure - I can't see any reason that it ought to be.

Reply:
I think we should re-write it using the default test project to avoid having to create extra test project.

Reply:
Agreed, and done.